storage/cloud-client/acl_test.py

```diff
@@ -39,37 +36,29 @@
 )
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
 def test_bucket():
-    """Yields a bucket that resets its acl after the test completes."""
-    bucket = storage.Client().bucket(BUCKET)
-    acl = google.cloud.storage.acl.BucketACL(bucket)
-    object_default_acl = google.cloud.storage.acl.DefaultObjectACL(bucket)
-    acl.reload()
-    object_default_acl.reload()
-    time.sleep(1)  # bucket ops rate limited 1 update per second
+    """Yields a bucket that is deleted after the test completes."""
+    bucket = None
+    while bucket is None or bucket.exists():
+        bucket_name = "acl-test-{}".format(uuid.uuid4())
```

Comment:
nit should this be snake case esp because the test blob is too?

Reply:
They are named like this elsewhere so I just repeated the pattern (see bucket lock)

Reply:
I just confirmed in the project. It seems we have pretty well used '-' elsewhere, so going to avoid starting a new trend.